cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -72,9 +71,20 @@ def test_cloud_compressed_view(request):
     set_and_test_view('Compare', 'Compressed View')
 
 
-def test_cloud_details_view(request):
-    set_and_test_view('Compare Mode', 'Details Mode')
+def set_and_test_mode(group_name, mode):
```

Comment:
**Required**: set_and_test_view function can be updated to use also mode_selector, as too much code duplication. Add a parameter for mode_selector, views_selector, as an option:
def set_and_test_mode(group_name, mode, selector_type='views_selector'):
     selected_view = getattr(inst_view.actions, selector_type).selected

Reply:
@rlbabyuk ok, that's wise